packages/engine-server/src/markdown/utils.ts

```diff
@@ -631,6 +513,128 @@ export class MDUtilsV4 {
   }
 }
 
+/** Contains functions that help dealing with mdast trees. */
+export class NodeUtils {
```

Comment:
node is a little confusing since it could mean many things. can we rename to `MdastUtils`?

Reply:
`mdast` is now called `remark`, so maybe something else. We already have a class called `RemarkUtils` though...